src/cascadia/TerminalSettingsEditor/Profiles.xaml

```diff
@@ -93,6 +89,24 @@ the MIT License. See LICENSE in the project root for license information. -->
         <PivotItem x:Uid="Profile_Appearance">
             <ScrollViewer>
                 <StackPanel HorizontalAlignment="Left">
+                    <TextBlock x:Uid="Profile_WindowHeader" Margin="0,0,0,10" Style="{StaticResource GroupingHeader}"/>
+                    <TextBox x:Uid="Profile_Padding"
+                             Text="{x:Bind State.Profile.Padding, Mode=TwoWay}"/>
+                    <Controls:RadioButtons x:Uid="Profile_ScrollbarVisibility"
+                                           ItemsSource="{x:Bind ScrollStateList, Mode=OneWay}"
+                                           SelectedItem="{x:Bind CurrentScrollState, Mode=TwoWay}"
+                                           ItemTemplate="{StaticResource EnumRadioButtonTemplate}"/>
+                    
+                    <TextBlock x:Uid="Profile_TextHeader" Style="{StaticResource GroupingHeader}"/>

```

Comment:
nit: I'd probably put the "text" settings first - I'd think that users change those the most often
(this could be done later)